spark/v3.2/spark/src/main/java/org/apache/iceberg/spark/Spark3Util.java

```diff
@@ -659,17 +658,7 @@ public String or(String leftResult, String rightResult) {
     }
 
     private static <T> String sqlString(List<org.apache.iceberg.expressions.Literal<T>> literals) {
-      return literals.stream().map(DescribeExpressionVisitor::sqlString).collect(Collectors.joining(", "));
-    }
-
-    private static String sqlString(org.apache.iceberg.expressions.Literal<?> lit) {
-      if (lit.value() instanceof String) {
-        return "'" + lit.value() + "'";
-      } else if (lit.value() instanceof ByteBuffer) {
-        throw new IllegalArgumentException("Cannot convert bytes to SQL literal: " + lit);
-      } else {
-        return lit.value().toString();
-      }
+      return literals.stream().map(Object::toString).collect(Collectors.joining(", "));
```

Comment:
This seems possibly incorrect or a deviation from the behavior before.

It looks like this code is calling Object.toString on `Literal<T>`, whereas before, the `toString` call came from `Literal<T>::value`. Is that the intended change?

Reply:
Thanks for your review. After testing, I think this is correct